app/models/user.rb

```diff
@@ -15,4 +18,16 @@ def normalize_queue_item_list
   def queued_video?(video)
     queue_items.find_by(video: video)
   end
+
+  def already_followed?(leader)
+    following_relationships.map(&:leader).include?(leader)
+  end
+
+  def is_the?(role)

```

Comment:
`is_the?` is a little bit strange method name, i would probably do something else like `has_role?`.